packages/dashboard/src/components/DashboardFilters/DashboardFiltersEditor/components/APIModal.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react'
+import { SharedFilter } from '../../../../types/SharedFilter'
+import Tooltip from '@cdc/core/components/ui/Tooltip'
+import Icon from '@cdc/core/components/ui/Icon'
+
+type APIModalProps = {
+  filter: SharedFilter
+  isSubgroup: boolean
+  isNestedDropdown: boolean
+  textSelector: string
+  valueSelector: string
+  updateAPIFilter: Function
+}
+
+const APIModal: React.FC<APIModalProps> = ({
+  filter,
+  isNestedDropdown,
+  updateAPIFilter,
+  valueSelector,
+  textSelector
+}) => {
+  const [APIEndpoint, setAPIEndpoint] = useState(filter.apiFilter?.apiEndpoint || '')
+  const [APIValueSelector, setAPIValueSelector] = useState(filter.apiFilter?.valueSelector || '')
+  const [APITextSelector, setAPITextSelector] = useState(filter.apiFilter?.textSelector || '')
+  const [APISubGroupValueSelector, setAPISubGroupValueSelector] = useState(
+    filter.apiFilter?.subgroupValueSelector || ''
+  )
+  const [APISubGroupTextSelector, setAPISubGroupTextSelector] = useState(filter.apiFilter?.subgroupTextSelector || '')
+  return (
+    <fieldset className='edit-block mb-1 px-3 cdc-open-viz-module'>
```

Comment:
anyway you can remove the edit-block and cdc-open-viz-module classes? would like to deprecate those in the future. 

Otherwise. I think you should reuse some other input components from the core package